jekyll/_cci2/creating-orbs.md

```diff
@@ -0,0 +1,332 @@
+---
+layout: classic-docs
+title: "Creating Orbs"
+short-title: "Creating Orbs"
+description: "Starting point for Creating CircleCI Orbs"
+categories: [getting-started]
+order: 1
+---
+
+Orbs are reusable packages of CircleCI configuration that you may share across projects, enabling you to create encapsulated, parameterized commands, jobs, and executors that can be used across multiple projects.
+
+This section describes how you can:
+
+* Use an existing orb in your configuration
+* Import an orb
+* Create an inline orb in your `config.yml` file
+* Create and publish an orb in the registry
+
+Orbs are made available for use in a configuration through the orbs key in the top level of your config.yml.
+
+## Using Existing Orbs
+
+CircleCI has made available a number of certified and 3rd party orbs that you may use in your configuration to reduce the time needed to get up and running using orbs in your configuration. By using CircleCI certified orbs, or 3rd-party orbs developed by CircleCI partners, you can be confident that these orbs have been developed and tested to ensure they work with the CircleCI platform.
+
+### Certified orbs vs. 3rd party Orbs
+
+Certified orbs are those that CircleCI has built or has reviewed and approved as part of the features of the CircleCI platform. Any project may use certified orbs in configuration version 2.1 and higher.
+
+3rd party orbs are those published by our customers and other members of our community. For you to publish orbs or for your projects to use 3rd party orbs, your organization must opt-in under SECURITY within the Organization Settings page under the section "Orb Security Settings" where an organization administrator must agree to the terms for using 3rd-party software.
+
+<aside class="notice">
+This setting can only be toggled by organization administrators.
+</aside>
+
+### Orb reference format
+
+Orb references have the following format shown below:
+
+```
+[namespace]/[orb]@[version]
+```
+
+### Semantic Versioning in Orbs
```

Comment:
I think this should have four hash signs, so it's under the `Orb Reference Format` heading along with Development Versions. Otherwise, it makes sense to drop a single line under `Orb Reference Format` that dev orbs are a bit different.

Reply:
fixed